src/main/scala/cromwell/engine/io/gcs/GcsFileSystem.scala

```diff
@@ -9,12 +9,13 @@ import java.util.{Collections, Set => JSet}
 import cromwell.engine.PathString
 
 import scala.concurrent.ExecutionContext
+import scala.util.Try
 
 object GcsFileSystem {
   import PathString._
-  def instance(interface: GoogleCloudStorage, root: String)(implicit executionContext: ExecutionContext) = {
+  def instance(interface: Try[GoogleCloudStorage], root: String)(implicit executionContext: ExecutionContext) = {
```

Comment:
IMO, `instance` implies this returns a singleton instance whereas this is creating a whole new one each time